app/controllers/api/v0/highlights_controller.rb

```diff
@@ -37,14 +39,12 @@ class Api::V0::HighlightsController < Api::V0::BaseController
   end
 
   def create
-    binding, error = bind(params.require(:highlight), Api::V0::Bindings::NewHighlight)
+    inbound_binding, error = bind(params.require(:highlight), Api::V0::Bindings::NewHighlight)
     render(json: error, status: error.status_code) and return if error
```

Comment:
it might be good to run this one thru rubocop.  it's correct to use and, but i remember rubocup once telling me on and statements like this to use && instead of and.  (it might be clearer to put the return on the next line).   sometimes and statements get lost in longer lines

Reply:
rubocop does complain but they have different precedences and as avdi grimm points out they are more control flow operators than boolean logic operators: http://www.virtuouscode.com/2010/08/02/using-and-and-or-in-ruby/

Reply:
just curious...why do you not like the multi-line way?  it's crystal clear w/ no chance of operator precedence ambiguity at all.

Reply:
too long.  `blah and return` is fairly common and I like the conciseness.